samples/Azure.Management.Storage/Azure.Management.Storage.csproj

```diff
@@ -11,11 +11,11 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Azure.ResourceManager" Version="1.0.0-alpha.20220310.5" />
+    <PackageReference Include="Azure.ResourceManager" Version="1.0.0-alpha.20220312.1" />
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Azure.Core" Version="1.23.0-alpha.20220310.5" />
+    <PackageReference Include="Azure.Core" Version="1.23.0-alpha.20220312.1" />
```

Comment:
Why do we need to upgrade the version of Azure.Core? Is it because the definition of `ResourceType` live in Azure.Core?

Reply:
I believe it's not because of some feature change. I have not figured out the root cause, but now if you publish an alpha version of Azure.ResourceManager, it relies on Azure.Core with the same alpha version number. And if we don't update the Azure.Core version here, compiler will complain that there is a downgrade of Azure.Core. We can probably remove the explicit dependency of Azure.Core here since Azure.ResourceMananger depends on Azure.Core.

Reply:
I have taken a look at recently changes. This is probably because that we change the dependency of Azure.Core from package reference to project reference.

Reply:
Ah, that explains it.